style.css

```diff
@@ -362,4 +362,106 @@
     border: 1px solid #c1c7d0;
     border-radius: 8px;
   }
+
+  .contact {
+    background-color: #6070ff;
+    border-top-left-radius: 80px;
+    background-image: url(./assets/contact-form-shapes-mobile.png);
+    margin-top: 114px;
+  }
```

Comment:
- [ ]  Kindly add a background image to this section that matches that on the design template
*Your design*
![Screenshot 2022-08-23 231900](https://user-images.githubusercontent.com/94551484/186276388-39a26cbb-6dfe-4306-bbe8-c06db0a5f360.png)

*Figma template*
![Screenshot 2022-08-23 231958](https://user-images.githubusercontent.com/94551484/186276404-35d9bd21-69c7-46b5-abc1-6e84315dbf3e.png)
.